modules/openapi-generator/src/main/resources/kotlin-client/libraries/multiplatform/ApiClient.kt.mustache

```diff
@@ -0,0 +1,17 @@
+package {{packageName}}
+
+import io.ktor.client.engine.HttpClientEngine
+import kotlinx.serialization.json.JsonConfiguration
+import {{packageName}}.apis.*
+
+class ApiClient(
+    baseUrl: String = "{{{basePath}}}",
+    httpClientEngine: HttpClientEngine? = null,
+    jsonConfiguration: JsonConfiguration = JsonConfiguration.Stable
+) {
+    {{#apiInfo}}
+    {{#apis}}
+    val {{{classVarName}}} = {{{classFilename}}}(baseUrl, httpClientEngine, jsonConfiguration)
```

Comment:
This may generate the following (invalid) code:
```kotlin
val package = PackageApi(baseUrl, httpClientEngine, jsonConfiguration)
```

Reply:
`package` is a reserved word and should be escaped. Did you perform a test that lead to the invalid example you provided? If so, could you share? I'd like to debug through to understand why package wasn't escaped.

Reply:
I can get you an openapi def that reproduces it, but it's rather large

Reply:
I can put together a minimal doc. I just figured I'd test the same as you if it was handy, but not a big deal if it's cumbersome.

Reply:
This is the thing:
https://gitlab.com/-/snippets/2007909 (github sadly errors when I try to upload it)

But it's _huge_

Reply:
The thing that's causing it is probably the
```json
"tags": [
  "Package"
]
```

Reply:
Where you able to find the issue that causes it's or should I try crafting a smaller file?

Reply:
Did a "hacky" solution, for now I just escape everthing. It works fine but it's not quite a best practice